src/Extractors/PhpCode.php

```diff
@@ -122,11 +122,11 @@ function ($match) {
                     case '\\':
                         return '\\';
                     case 'x':
-                        return chr(hexdec(substr($match[0], 1)));
+                        return chr(hexdec(substr($match[1], 1)));
                     case 'u':
-                        return self::unicodeChar(hexdec(substr($match[0], 1)));
+                        return self::unicodeChar(hexdec(substr($match[1], 1)));
                     default:
-                        return chr(octdec($match[0]));
+                        return chr(octdec($match[1]));
```

Comment:
Why did you change this?

Reply:
`$match[0]` is something like x20 which cannot be octdev (it raise a warning), `$match[1]` don't have the prefix.